lib/services/service_brokers/v2/client.rb

```diff
@@ -79,15 +79,35 @@ def fetch_service_instance_state(instance)
       result.merge(parsed_response.symbolize_keys)
     end
 
+    def create_service_key(key, arbitrary_parameters: {})
```

Comment:
Instead of adding a method like this, can we just add a little more logic to the `bind` method? There is so much duplication between `bind` and `create_service_key`, I think it's better to have one method, and it choose which orphan mitigation technique to use. Something like this maybe:
```
    ...
    rescue Errors::ServiceBrokerApiTimeout, Errors::ServiceBrokerBadResponse => e
      if binding.respond_to? :app_guid
        @orphan_mitigator.cleanup_failed_bind(@attrs, binding)
      else
        @orphan_mitigator.cleanup_failed_key(@attrs, binding)
      end
      raise e
    end
```